include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
If I make more commits to my branch does the show up in this pull request?  Do you want me to update my pull request? I won't be offended if someone else gets the fix in place. I was just trying to be helpful.